source/components/index.tsx

```diff
@@ -2,7 +2,22 @@ import * as React from 'react';
 import ReactDOM from 'react-dom';
 
 import 'semantic-ui-css/semantic.min.css';
-
+import {BrowserRouter, Route, Routes} from 'react-router-dom';
 import Popup from './Popup';
+import Send from '../screens/Send';
+import Home from '../screens/Home';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route element={<Popup />}>
```

Comment:
Good job on this one. Does it work as a stack of screens, or every time `/send` view gets closed, `Home` gets re-rendered?